region/lowernorfair/east/Three Musketeers' Room.json

```diff
@@ -146,7 +146,8 @@
           "limit": 10
         }},
         {"partialRefill": {"type": "Super", "limit": 4}}
-      ]
+      ],
+      "resetsObstacles": ["A"]
```

Comment:
The kihunters are A. Not sure if this should be clears or resets. Since youre heatproof maybe clears would be better?

Reply:
Done.